### ISSUE TITLE
Text cannot be enlarged to 200% (WCAG 1.4.4a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1228]
When the text is enlarged to 200%, the page content is cut off and is not fully visible. The size of layout areas can be adapted to different output formats with the help of media queries and flexbox, so that content in the available space increases, decreases or breaks and reacts to zoom levels. The properties width and max-width for labels and input fields should be set so that they adapt to the output format.

1. https://slidewiki.org/ 
2. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
3. a. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
b. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
4. https://slidewiki.org/search?keywords=accessibility&sort=score 
5. https://slidewiki.org/user/vivien.zschammer/recommendations 
6. https://slidewiki.org/addDeck 
7. https://slidewiki.org/contactus 
8. https://slidewiki.org/deck/127411/test/deck/127411 
9. https://slidewiki.org/accessibility 
10. a. https://slidewiki.org/user/vivien.zschammer/settings/profile
b. https://slidewiki.org/user/vivien.zschammer/stats

**Comment by aoelen** [comment]
I wasn't able to reproduce this issue. To me, the slide scales well at 200%. The nav bar does change indeed (some elements are hidden). But they are still accessible via the hamburger menu on the left. 

If there are any other problem, could you describe what is wrong exactly?

**Comment by VivienDZB** [comment]
You are right indeed. When zooming the whole page, most of it looks fine. I still see some issues with the presenation view page, on which the content can´t be enlarged to 200% (https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/?language=en#/slide-437649-3). 

Also when this criterion is tested, it includes testing if it is possible to zoom text only, without it being cut off:

"Therefore you need to open page in Firefox, use the Web Developer Toolbar to set the browser window size to 1280x768 (change size> ...). Activate the option "zoom only text" under View> Zoom. Zoom the page to 200%. Check whether all texts are still visible and legible. They should be displayed completely without overlaps or parts of text being cut off. The line height should adapt to the font size. Check whether continuous texts can be read without horizontal scrolling. The mere presence of the cross-scroll bar is not assessed negatively. Rather, it is important that all continuous texts can be read without constant horizontal scrolling. The horizontal scroll bar is set so that the content of the widest column is displayed in the window. Can the body text be read in this orientation without moving the horizontal scroll bar left and right for each individual line?" (BITV 1.4.1a).

I still noticed issues (overlapping text and/or text cut off) on some pages regarding those criteria:
https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit
https://slidewiki.org/user/vivien.zschammer/settings/profile


I also just noticed that the Anmelden dialogue appears twice at 200%. I zoomed out for the screenshot:

![grafik](https://user-images.githubusercontent.com/48785136/81386897-107a9b00-9116-11ea-9875-ca7bd4992ca9.png)

(The top one is better btw when it comes to zooming because user name and password are completely visable. ;) )

**Comment by aoelen** [comment]
For most pages we improve the responsiveness and therefore also the ability to zoom. But you are right, at a few locations text is not fully visible, but I think this is a minor issue (a quick check from my side showed that it works for most pages now). So I suggest to close this issue. 

I will have a look at the double sign in dialog. That looks like a seperate bug.

**Comment by aoelen** [comment]
I was able to reproduce the issue with the sign in dialog: https://github.com/slidewiki/slidewiki-platform/issues/1272

Closing this one.

**Comment by VivienDZB** [comment]
I still have some issues of overlapping text or elements when zooming to 200%:

### Home - https://slidewiki.org/
When the text is enlarged to 200%, the search field and the buttons "Meine Präsentationen" and "Erfahren Sie mehr" overlap.
![grafik](https://user-images.githubusercontent.com/48785136/95976350-a6c69180-0e17-11eb-8e53-97b89fb1384c.png)

### Search - https://slidewiki.org/search?keywords=accessibility&sort=score
![grafik](https://user-images.githubusercontent.com/48785136/95976481-dbd2e400-0e17-11eb-927d-30a64d8191f0.png)
Maybe it is possible to place the form elements one below the other. 
### Editor - https://slidewiki.org/deck/131649-1/_/slide/856768-2/856768-2:2/edit
![grafik](https://user-images.githubusercontent.com/48785136/95976585-0755ce80-0e18-11eb-9982-ae8df936ba23.png)

### Presentation view - https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/437650-5/?language=en#/slide-437650-5
Text overlaps at 200%.

**Comment by aoelen** [comment]
I fixed this in several locations, but unfortunately it isn't easy to do this for everything. For example, fixing the overlapping buttons in the editor requires a redesign of this element (media queries cannot be used since only the text is zoomed, and not the browser window). 

Fixes:
![image](https://user-images.githubusercontent.com/1567948/97002910-ef2e2f80-153a-11eb-9f6b-96c0477aa45b.png)

![image](https://user-images.githubusercontent.com/1567948/97002945-ffdea580-153a-11eb-83f0-d34d858911c5.png)

![image](https://user-images.githubusercontent.com/1567948/97002976-0a993a80-153b-11eb-9962-f466dc7a8db2.png)
